CMakeLists.txt

```diff
@@ -74,6 +74,7 @@ execute_process(COMMAND git log --oneline
 
 STRING(REGEX REPLACE "\n" "" git_ver "${git_ver}")
 add_definitions(-DHASH="${git_ver}")
+add_definitions(-fgnu89-inline)
```

Comment:
Probably should [fix the warning](https://stackoverflow.com/a/12749120) by adding declarations instead of rely on GCC extensions. What if we want to build with llvm?

Reply:
True, @nascs